specs/engage-digital_openapi3.yaml

```diff
@@ -3181,6 +3181,23 @@ paths:
           required: false
           schema:
             type: string
+        - description: >-
+            To filter on the intervention status. Possible values are:
+             - active (open but not deferred)
+             - closed
+             - deferred
+             - opened
+             - reopened

```

Comment:
I added the list of values in description so it's available in postman and also to add an explanation about what "active" means.

Reply:
:+1: